### ISSUE TITLE
Failure to adequately address how accessibility "Supports Diversity and Inclusion"

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-bcase#32]
Rewrote "Brand Enhancement" section to address concern of @vmmiller. Please comment if revision sufficiently addresses this concern.

**Comment by yatil** [comment]
(follow up to #25)

**Comment by sharronrush** [comment]
Hearing no further objection in 18 days, am closing this issue.
